frontend/src/components/Calendar/Day/Day.tsx

```diff
@@ -107,21 +109,36 @@ export function Day(props: {
           (60 - event.beginDate.getSeconds()) / 3600;
       }
     }
-    eventDate.push(startTime);
+    eventsBeginTime.push(startTime);
   });
 
   return (
-    <div id={`${day}`} className="blockDisplay">
+    <div
+      id={`${day.toLocaleDateString('en-EN', {
+        weekday: 'long',
+        month: 'long',
+        day: 'numeric',
+      })}`}
```

Comment:
Utiliser la langue de trad + pas la peine d'encapsuler un string dans un string

```suggestion
      id={day.toLocaleDateString(i18n.language, {
        weekday: 'long',
        month: 'long',
        day: 'numeric',
      })}
```

Reply:
C'est pas casse-pied d'avoir un id changeant en fonction de la langue comme id ?